packages/reporter/test/mocha.opts

```diff
@@ -1,3 +1,3 @@
---require '@packages/web-config/node-register'
+--require ../web-config/node-register
```

Comment:
curious why you can't use a symlink here?

Reply:
I thought the tests were failing otherwise, but I think I just realized that the red output was expected, so I'll change this back.